src/com/homework/MyList.java

```diff
@@ -0,0 +1,15 @@
+package com.homework;
+
+public interface MyList <E> {
+    boolean add(E e);
+
+    void remove(int index);
+
+    void clear();
+
+    int size();
+
+    E get(int index);
+}
+
+//add, remove, clear, size, get;
```

Comment:
Do you need these comments?
Please add a blank line at the end of file

Reply:
not really - deleted, left the line